### DISCUSSION TITLE
Job is failing on warning

### DISCUSSION THREAD

**Discussion by LeGmask** [laravel/framework#42989] [Q&A] [answered]
Heyy, is there a way to disable failing when triggering a warning ?

Ty by advance
Gmask

**Comment by henzeb** [comment, marked as answer]
Are you sure it's a warning, and not something else?

You can change the php `error_reporting(E_ALL);` to `error_reporting(E_ALL &~ E_USER_WARNING);` or something similar.

This will ignore any warnings.

**Reply by LeGmask** [reply]
Yes i'm sure it's a warning since when I'm running my lib outside of a job I have the error but not as an error as a warning ...
Yeah the error_reporting is working thank you
